### ISSUE TITLE
Images broken if previewing a Post without updating it first

### ISSUE THREAD

**Issue by ngoctr** [wordpress-mobile/gutenberg-mobile#2159]
**Describe the bug**
While we were working on [this](https://github.com/wordpress-mobile/WordPress-iOS/issues/13803), we had noticed an issue where previewing in the Block Editor an unsaved Post will make recent added images being broken.

**To Reproduce**
1. From WordPress app, Go to **Site -> Posts** then select a Post to edit. Switch to Block Editor if you're on the classic editor.
2. Add an Image block, then tap **ADD IMAGE** and choose **Free Photo Library**
3. Search and select an image of your choice
4. Wait for the image to be uploaded
5. Tap the ... menu at the top-right corner, select Preview, you will notice images to be broken

**Expected behavior**
User should see the image they have added to the Post

**Screenshots**
![image](https://user-images.githubusercontent.com/17534738/79404139-2e038b80-7fd4-11ea-9921-f9e6f2e2528d.png)

**Smartphone (please complete the following information):**
 - Device: [iPhone, iPad]
 - OS: [iOS]
 - Version [13.4]

**Additional context**
N/A

**Comment by maxme** [comment]
> In my opinion, since the image has not been uploaded yet so the preview won't be able to show the image. However, the Preview button is there, and it user would expect to see images when previewing a Post.

I agree. In wpandroid (with classic or the block editor), we show a toast message ("You are currently uploading media. Please wait until this completes.") when the Preview button is pressed during an image upload. 

Ref. https://github.com/wordpress-mobile/WordPress-Android/blob/a3988b1b990862b2e7ee16df6737c449a43349f1/WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java#L1112

I think the same behavior should be implemented on wpios (I thought this was part of the preview revamp project cc @diegoreymendez @yaelirub)

**Comment by yaelirub** [comment]
Thanks @maxme ! I'll add this to our backlog
